configuration/configuration.go

```diff
@@ -86,6 +87,11 @@ const (
 	// by hosted node services. When not set, defaults to false.
 	SkipGethAdminEnv = "SKIP_GETH_ADMIN"
 
+	// GethHeaders is an optional environment variable
+	// of a comma-separated list of key:value pairs to apply
+	// to geth clients as headers. When not set, defaults to []
+	GethHeaders = "GETH_HEADERS"
```

Comment:
let's name this field `GethHeadersEnv` :)

Reply:
Done!